redash/handlers/static.py

```diff
@@ -6,14 +8,21 @@
 from redash.handlers.authentication import base_href
 from redash.handlers.base import org_scoped_rule
 from redash.security import csp_allows_embeding
+from jinja2.exceptions import TemplateNotFound
+
+logger = logging.getLogger(__name__)
 
 
 def render_index():
-    if settings.MULTI_ORG:
-        response = render_template("multi_org.html", base_href=base_href())
-    else:
-        full_path = safe_join(settings.STATIC_ASSETS_PATH, 'index.html')
-        response = send_file(full_path, **dict(cache_timeout=0, conditional=True))
+    try:
+        if settings.MULTI_ORG:
+            response = render_template("multi_org.html", base_href=base_href())
+        else:
+            full_path = safe_join(settings.STATIC_ASSETS_PATH, 'index.html')
+            response = send_file(full_path, **dict(cache_timeout=0, conditional=True))
+    except TemplateNotFound as e:
+        logger.exception("%s is not found", e.name)
+        response = render_template("error.html", error_message='Index page is not found.')
```

Comment:
Let's make the message even friendlier:

```python
if current_app.debug: # need to make sure current_app is imported
    message = "Missing template file ({}). Did you build the frontend assets with npm?".format(e.name)
else:
    message = "Error Rendering Page."
```

(and use this for the `error_message`)